src/library/mixxxlibraryfeature.cpp

```diff
@@ -176,12 +179,16 @@ void MixxxLibraryFeature::searchAndActivate(const QString& query) {
     activate();
 }
 
-#ifdef __ENGINEPRIME__
 void MixxxLibraryFeature::bindSidebarWidget(WLibrarySidebar* pSidebarWidget) {
-    // store the sidebar widget pointer for later use in onRightClick
+    // store the sidebar widget pointer for later use in onRightClick and
+    // updating track count
     m_pSidebarWidget = pSidebarWidget;
 }
-#endif
+
+void MixxxLibraryFeature::slotUpdateTrackCount() {
+    m_trackCount = m_pLibraryTableModel->rowCount();
+    m_pSidebarWidget->repaint();
```

Comment:
```suggestion
    if (m_pSidebarWidget) {
        m_pSidebarWidget->repaint();
    }
```
otherwise it would crash during auto-scan on startup when the sidebar hasn't been created, yet.